File: Script/Fibonacci_iterativo/Fibonacci.c
#include <stdio.h>
#include <stdlib.h>

int fibonacci(int n) {
    if (n <= 1) return n;
    int a = 0, b = 1, c;
    for (int i = 2; i <= n; i++) {
        c = a + b;
        a = b;
        b = c;
    }
    return b;
}

int main(int argc, char *argv[]) {
    if (argc != 2) {
        printf("Uso: %s <numero>\n", argv[0]);
        return 1;
    }
    int n = atoi(argv[1]);
    printf("Fibonacci(%d) = %d\n", n, fibonacci(n));
    return 0;
}

File: Script/Fibonacci_iterativo/Fibonacci.py
import sys

def fibonacci(n):
    if n <= 1:
        return n
    a, b = 0, 1
    for _ in range(2, n+1):
        a, b = b, a + b
    return b

if __name__ == "__main__":
    if len(sys.argv) != 2:
        print(f"Uso: python fibonacci_py.py <numero>")
        sys.exit(1)
    n = int(sys.argv[1])
    print(f"Fibonacci({n}) = {fibonacci(n)}")
